ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -19,6 +19,29 @@ import UIKit
 @objc(MSFBottomCommandingController)
 open class BottomCommandingController: UIViewController {
 
+    /// View controller that will be displayed below the bottom commanding UI.
+    @objc public var contentViewController: UIViewController? {
```

Comment:
should it be part of the init?

Reply:
For Office purposes we need the settable property, but I also added the init because we should probably encourage giving us the content VC right away.